manifests/i/IObit/Uninstaller/15.0.0.10/IObit.Uninstaller.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: IObit.Uninstaller
+PackageVersion: 15.0.0.10
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /NORESTART /Install
+  SilentWithProgress: /SILENT /NORESTART /Install
+  Log: /LOG "<LOGPATH>"
+UpgradeBehavior: install
+ProductCode: IObitUninstall
+AppsAndFeaturesEntries:
+- DisplayName: IObit Uninstaller 15
+  Publisher: IObit
+  ProductCode: IObitUninstall
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Maybe try `x64` to see if the unattend error goes away?

Since having `x86` here will make the pipelines use x86 version of Windows 10 rather than x64.

---

Usually it will work just fine with x86, but I could try this.